toolkit/tools/imagecustomizer/README.md

```diff
@@ -46,7 +46,8 @@ Disadvantages:
 3. Install prerequisites: `qemu-img`, `rpm`, `dd`, `lsblk`, `losetup`, `sfdisk`,
```

Comment:
Chris, do we need to add docs for the new api surface and its dependencies?

Reply:
Good catch! Thanks. Fixed.